.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and test
+
+on:
+  push:
+    branches: [ "develop", "develop_1.x" ]
+  pull_request:
+    branches: [ "develop", "develop_1.x" ]
+
+jobs:
+  define-ios-device:
+    name: Get iOS simulator device to run iOS tests on
+    runs-on: macos-latest
+    outputs:
+      device: ${{ steps.ios.outputs.device }}
+    steps:
+      - id: ios
+        run: echo "device=`xcrun xctrace list devices 2>&1 | grep -oE 'iPhone.*?[^\(]+' | head -1 | awk '{$1=$1;print}' | sed -e "s/ Simulator$//"`" >> "$GITHUB_OUTPUT"
+
+  build:
+    name: Build and Test mamba and mambaTVOS
+    runs-on: macos-latest
+    needs: define-ios-device
+    strategy:
+      matrix:
+        target:
```

Comment:
this matrix strategy stuff is really cool and new to me 😄 . Very nice.

Reply:
Yeah, was new to me too! I was just doing a little reading on their docs (trying to figure out how to run multiple targets) and found it... GitHub Actions is just a great tool.